compiler_opt/es/blackbox_evaluator.py

```diff
@@ -159,3 +160,17 @@ def set_baseline(self, pool: FixedWorkerPool) -> None:
                        f' got {len(futures)}')
 
     self._baseline = futures[0].result()
+
+  def get_rewards(
+      self, results: list[concurrent.futures.Future]) -> list[float | None]:
+    rewards = []
+
+    for result in results:
+      if result.exception() is not None:
+        raise result.exception()
+
+      # pylint: disable=protected-access
```

Comment:
why not make `compilation_runner._calculate_reward` "public" - just also add a test for it (pro'lly that a 0 baseline doesn't Inf)